supporting_files/5gmag-rt.conf

```diff
@@ -51,8 +51,8 @@ gw: {
   http_server: {
     uri: "http://0.0.0.0:3020/";
     api_path: "gw-api";
```

Comment:
I will create an issue for that so we don't forget to rename `gw-api` as well once `rt-wui` has been adjusted